### ISSUE TITLE
[BITV] 9.2.2.1/3.1 - Rich workspace autohides toolbar during usage

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/text#3862]
- [ ] Fix
- [ ] Backport

When editing text, the formatting buttons are not continuously visible but close unexpectedly after a very short timeout, even when user interactions are taking place (such as selecting a heading level from dropdown menu for headings). Editing text in the "div" with "contenteditable=true" necessitates moving the cursor back to the controls. A lot more time is needed then is currently given. When dropdown menus for heading styles or insert emoi are opened they auto-close after a very short time. This makes the editing controls practically unusable especially with the keyboard but also for pointer users that need more time. (2)

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222247240-e7aae9a1-15b6-439e-8e82-24932974a718.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-2c3b5688b6-v3-n1

</details>

- [ ] 9.2.2.1/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-2c3b5688b6-v10-n1

**Comment by juliushaertl** [comment]
This should have been fixed by https://github.com/nextcloud/text/pull/2717 needs retesting on our side

**Comment by datenangebot** [comment]
Looks good to me. Will close.
